wasm.md

```diff
@@ -1260,8 +1260,8 @@ The value is fetched from the "effective address", which is the address given on
                         | IValType "." LoadOp Int
  // ---------------------------------------------
 
-    syntax Signedness ::= "Signed" | "Unsigned"
- // -------------------------------------------
+    syntax Signedness ::= "SignedI" | "UnsignedI"
```

Comment:
What good does this change do? Is it that it's supposed to represent ints? If so, I don't see why, these two tokens could equally well be used to represent signedness of floats right? They are just an argument to the `load` function.

Reply:
Because There is exactly the same production defined in module BYTES (you can checkout function BYTES2INT and see the third parameter).
If don't do this there will be multi-define error.

Reply:
Alright, then just delete this rule here, and we can use the one from BYTES instead.

Reply:
In BYTES module they have special klabels for theit signedness and is not really good to be used here..